typescript/src/enums.ts

```diff
@@ -244,3 +244,45 @@ export enum PURCHASES_ARE_COMPLETED_BY {
    */
   REVENUECAT = "REVENUECAT",
 }
+
+/**
+ * Configuration option that specifies that your app will complete purchases.
+ * @public
+ */
+export type PURCHASES_ARE_COMPLETED_BY_MY_APP = {
+  type: PURCHASES_ARE_COMPLETED_BY_TYPE.MY_APP;
+
+  /**
+   * The version of StoreKit that your app is using to make purchases. This value is ignored
+   * on Android, so if your app is Android-only, you may provide any value.
+   */
+  storeKitVersion: STOREKIT_VERSION;
+};
+
+/**
+ * Allows you to specify whether you want RevenueCat to complete your app's purchases
+ * or if your app will do so.
+ *
+ * You can configure RevenueCat to complete your purchases like so:
+ * ```typescript
+ * Purchases.configure({
+ *  apiKey: "123",
+ *  purchasesAreCompletedBy: PURCHASES_ARE_COMPLETED_BY.REVENUECAT,
+ * });
+ * ```
+ *
+ * You can specify that purchase are completed by your app like so:
+ * ```typescript
+ * Purchases.configure({
+ *  apiKey: "123",
+ *  purchasesAreCompletedBy: {
+ *    type: PURCHASES_ARE_COMPLETED_BY.MY_APP,
+ *    storeKitVersion: STOREKIT_VERSION.STOREKIT_1
+ *  },
+ * });
+ * ```
+ * @public
+ */
+export type PURCHASES_ARE_COMPLETED_BY =
```

Comment:
Same here about the all case naming. I would do `PurchasesAreacompletedBy` here?

Reply:
Updated! :D